content/cloud-docs/users-teams-organizations/organizations.mdx

```diff
@@ -13,179 +13,188 @@ description: >-
-See [Teams][] for more information.
+#### Variable Sets
 
-### Users
+View all of the available variable sets and their variables. Users with [`read and write variables` permissions](/cloud-docs/users-teams-organizations/permissions#general-workspace-permissions) can also create variable sets and assign them to one or more workspaces.
 
-The users page allows organization owners to invite new Terraform Cloud users into the organization, cancel invitations, and remove existing members.
+Variable sets let you reuse the same variables across multiple workspaces in the organization. For example, you could define a variable set of provider credentials and automatically apply it to several workspaces, rather than manually defining credential variables in each. Changes to variable sets instantly apply to all appropriate workspaces, saving time and reducing errors from manual updates.
 
-The list of users is split into two tabs: one for currently active users, and one for invited users who haven't accepted their invitations yet. If the lists are large, you can filter them by username or email using the search field at the top. For active users, the list includes usernames, email addresses, avatar icons, two-factor authentication status, and current team memberships.
+Refer to the [variables overview](/cloud-docs/workspaces/variables) documentation for details about variable types, scope, and precedence. Refer to [managing variables](/cloud-docs/workspaces/variables/managing-variables) for details about how to create and manage variable sets.
 
-![Screenshot: the users list, showing information for the currently active users.](/img/docs/org-users.png)
+### Integrations
 
-To invite a user to an organization, click the "Invite a user" button on the "Users" page. You will be asked to provide an email address and an optional list of teams. If the user accepts the emailed invitation, they will be automatically added to the specified teams.
+#### Cost Estimation
 
-![Screenshot: the "Invite a user" dialog, which includes fields for email address and teams.](/img/docs/org-users-invite.png)
+-> **Note:** Cost estimation is a paid feature, available as part of the **Team & Governance** plan. Refer to [Terraform pricing](https://www.hashicorp.com/products/terraform/pricing) for details.
 
-User invitations are always sent by email; you cannot invite someone using their Terraform Cloud username.
+Enable and disable the [cost estimation](/cloud-docs/cost-estimation) feature for all workspaces.
 
--> **Note:** All permissions in Terraform Cloud are managed via teams. Although users can join an organization without belonging to any teams, they won't be able to do anything until they are also added to a team. ([More about permissions.](/cloud-docs/users-teams-organizations/permissions))
+#### Policies
 
-[permissions-citation]: #intentionally-unused---keep-for-maintainers
+This is a deprecated interface for managing Sentinel policies. Use the policy sets page instead.
```

Comment:
```suggestion

```
Is this even worth keeping if it is deprecated?

Reply:
That's a fair point, Dane. It's still in the UI, so I think we do need to document it so folks aren't confused. When they take it out of the UI, I can take it out of the docs!